### ISSUE TITLE
No information

### ISSUE THREAD

**Issue by Yo-kai-Sei-shin-kage** [nitinkgp23/fr-locate#2]
Your README file is completely bare bones and doesn't explain how this tool is actually working. From looking at the code, it uses facebook api platform for a custom app. But this won't work unless friends actually allow the app to access their information on facebook. You need more information about how the tool works in your README file and not just an infograph.

**Comment by JavaRockstar** [comment]
I agree this tool could do with a lot more information and documentation i am guessing its a browser sided tool that connects to facebook api to provide the infomation back.

Any insight would be great really nice project but very confusing READ.md
